File: MiniDOMBase/DMD/Anagrafica/Postazioni/Contatori/ValoriRegistri.vb
Imports minidom
Imports minidom.Sistema
Imports minidom.Databases
Imports minidom.Anagrafica
Imports minidom.Internals

Namespace Internals

    'Classe globale per l'accesso agli uffici
    Public Class CValoriRegistriClass
        Inherits CModulesClass(Of ValoreRegistroContatore)

        Friend Sub New()
            MyBase.New("modPostazioniRegistri", GetType(ValoreRegistroCursor), 0)
        End Sub


    End Class
End Namespace

File: MiniDOMBase/DMD/Anagrafica/Postazioni/Contatori/ValoriRegistri.cs
using System;
using DMD.XML;
using DMD;
using DMD.Databases;
using DMD.Databases.Collections;
using static minidom.Sistema;
using static minidom.Anagrafica;

namespace minidom.repositories
{

    /// <summary>
    /// Repository di oggetti di tipo <see cref="ValoreRegistroContatore"/>
    /// </summary>
    [Serializable]
    public class CValoriRegistriClass 
        : CModulesClass<Anagrafica.ValoreRegistroContatore>
    {

        /// <summary>
        /// Costruttore
        /// </summary>
        public CValoriRegistriClass() 
            : base("modPostazioniRegistri", typeof(Anagrafica.ValoreRegistroCursor), 0)
        {
        }
    }
}